### DISCUSSION TITLE
bunch of created App on deployed Policy Pipeline

### DISCUSSION THREAD

**Discussion by leslierd** [Azure/CanadaPubSecALZ#372] [Q&A] [answered]
![image](https://github.com/Azure/CanadaPubSecALZ/assets/67053751/aa7f504d-3729-47c4-adc1-2870bbc7a2a0)
I would like to know why those apps are created for the Logging Subscription and their purpose when I run the policy-ci pipeline? 

thank you

**Comment by SenthuranSivananthan** [comment, marked as answer]
@leslieRD These are managed identities created for each policy set assignment.  The managed identities are used to automatically remediate policy drifts through deployIfNotExist policy type.

Managed Identities are assigned at the top level management group (and inherited down to all subscriptions in that MG scope) by default.  This assignment is based on where the policy set is assigned.  In the default case, it is at the top level management group.

**Reply by leslierd** [reply]
Thank you!
